src/effects/builtin/dtmfgen/dtmfgenerator.h

```diff
@@ -10,6 +10,10 @@ struct DtmfSettings
     static constexpr char DefaultSequence[] = "audacity";
     static constexpr double DefaultDutyCycle = 55.0;
     static constexpr double DefaultAmplitude = 0.8;
+    static constexpr double AmplitudeMin = 0.0;
```

Comment:
In DtmfBase.h, `Amplitude` has min 0.001. Not sure what's better, please check with @LWinterberg.